### DISCUSSION TITLE
My account flagged

### DISCUSSION THREAD

**Discussion by emrahyilm4z** [community/community#147819] [Accessibility]
### Select Topic Area

General

### GitHub Feature Area

VSCode

### Body

Hi
I can't use VS Code with Github. My account flagged. I opened a ticket and after 45 days there has still been no response. 45 DAYSS

**Comment by Olsoman** [comment]
i got the same issue as you, cant even submit a ticket because of some random "degenerative filter" for filtering accounts, and support is only in the paid tier, so i guess no help for us..
